charts/streams-app/templates/deployment.yaml

```diff
@@ -103,6 +103,13 @@ spec:
             - name: "{{ .Values.configurationEnvPrefix }}_ERROR_TOPIC"
               value: {{ .Values.streams.errorTopic | quote }}
           {{- end }}
+          {{- range $key, $value := .Values.secrets }}
+            - name: "{{ $key }}"
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ $key | lower}}"
```

Comment:
Why lower? Quote?

Reply:
Lower, because keys are only allowed to be lowercase and have `.` and `-` in it. The quotes don't change anything in the test: https://gitlab.bayer.com/sih/kafka-deployments/alignment-deployment/-/jobs/671556#L286

Reply:
I feel safer with quote additionally